src/test/java/com/parkhomovsky/bookstore/controller/CategoryControllerTest.java

```diff
@@ -0,0 +1,283 @@
+    private static final CategoryDto FANTASY_DTO = new CategoryDto()
+            .setId(3L)
+            .setName("Fantasy")
+            .setDescription("Books about mystery worlds and unbelievable");
+    private static final CategoryRequestDto FANTASY_REQUEST_DTO =
+            new CategoryRequestDto()
+                    .setName("Fantasy")
+                    .setDescription("Books about mystery worlds and unbelievable");
+    private static final CategoryRequestDto INVALID_FANTASY_REQUEST_DTO =
+            new CategoryRequestDto()
+                    .setName("")
+                    .setDescription("Books about mystery worlds and unbelievable");
+    private static final BookDtoWithoutCategoryIds RESPONSE_FICTION_BOOK_DTO_WITHOUT_CATEGORY_ID =
+            new BookDtoWithoutCategoryIds()
+            .setId(1L)
+            .setTitle("Test Book")
+            .setAuthor("Test Author")
+            .setIsbn("1315616")
+            .setPrice(new BigDecimal("14.56"))
+            .setDescription("Test book description")
+            .setCoverImage("https://URL");
+    private static final BookDtoWithoutCategoryIds RESPONSE_KOBZAR_BOOK_DTO_WITHOUT_CATEGORY_ID =
+            new BookDtoWithoutCategoryIds()
+            .setId(3L)
+            .setTitle("Kobzar")
+            .setAuthor("Shevchenko")
+            .setIsbn("123456789-123")
+            .setPrice(new BigDecimal("15"))
+            .setDescription("Good UA book")
+            .setCoverImage("https://URL");
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @BeforeAll
+    static void beforeAll(@Autowired WebApplicationContext applicationContext) {
+        mockMvc = MockMvcBuilders
+                .webAppContextSetup(applicationContext)
+                .apply(springSecurity())
+                .build();
+    }
+
+    @WithMockUser(username = "admin", roles = {"ADMIN"})
+    @Test
+    @Sql(
+            scripts = "classpath:db-scripts/clear-books_connections-tables.sql",
+            executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD
+    )
+    @DisplayName("Test creating category with a valid request body")
+    void create_validRequestBody_returnCategoryDto() throws Exception {
+        MvcResult mvcResult = mockMvc.perform(post("/categories")
+                        .content(objectMapper.writeValueAsString(FANTASY_REQUEST_DTO))
+                        .contentType(MediaType.APPLICATION_JSON)
+                )
+                .andExpect(status().isCreated())
+                .andReturn();
+
+        CategoryDto actual = objectMapper.readValue(
+                mvcResult.getResponse().getContentAsString(), CategoryDto.class);
+
+        assertTrue(EqualsBuilder.reflectionEquals(FANTASY_DTO, actual, "id"));
+    }
+
+    @WithMockUser(username = "admin", roles = {"ADMIN"})
+    @Test
+    @Sql(
+            scripts = "classpath:db-scripts/clear-books_connections-tables.sql",
```

Comment:
so, in one class you created constants but in this class decided not to do it. try to be consistent and if you chose one approach, then stick the common pattert throughout all application